localoam/index.js

```diff
@@ -0,0 +1,52 @@
+var http = require('http');
+var filed = require('filed');
+var glob = require('glob');
+var fs = require('fs');
+
+var master = {
+  'nodes': [
+    {
+      'locations': [
+        {
+          'type': 'localoam',
+          'url': 'http://localoam:4999'
+        }
+      ]
+    }
+  ]
+};
+
+function sendJSON (json, res) {
+  res.setHeader('Content-Type', 'application/json');
+  res.write(json);
+  res.end();
+}
+
+var server = http.createServer(function (req, res) {
+  var url = req.url;
+  if (url === '/list') {
```

Comment:
This and `sendJSON` cause me to wonder why you didn't pull in Express as a dependency.